crates/miden-objects/src/note/note_tag.rs

```diff
@@ -460,46 +442,15 @@ mod tests {
         let network_account_id = AccountId::try_from(NETWORK_ACCOUNT_INT).unwrap();
 
         // Expected network tag with LOCAL_EXECUTION_WITH_ALL_NOTE_TYPES_ALLOWED.
-        let expected_network_local_tag = NoteTag(0b11101010_10110011_00000000_00000000);
+        let _expected_network_local_tag = NoteTag(0b11101010_10110011_00000000_00000000);
```

Comment:
Why do we need these lines? It seems like we are declaring a variable but not using it.